util/tracing/Makefile

```diff
@@ -30,8 +31,10 @@ install: trace_to_csv trace_to_chrome trace_to_influxdb
 	mv trace_to_csv $(INSTALL_PREFIX)
 	mv trace_to_chrome $(INSTALL_PREFIX)
 	mv trace_to_influxdb $(INSTALL_PREFIX)
-	ln -f -s launch-fedsd.sh $(INSTALL_PREFIX)/fedsd
-	chmod +x launch-fedsd.sh
+	cp ./visualization/fedsd.py $(INSTALL_PREFIX)
+	cp ./visualization/fedsd_helper.py $(INSTALL_PREFIX)
```

Comment:
This file shouldn't end up on the path but in a `lib` directory. However, Python files are pretty small, so why not merge them?

Reply:
I explicitly asked about this.
Will do.